ou_dedetai/cli.py

```diff
@@ -78,6 +69,13 @@ def run_installed_app(self):
     def stop_installed_app(self):
         self.logos.stop()
 
+    def wine(self):
+        wine.run_wine_proc(
+            self.conf.wine64_binary,
+            self,
+            exe_args=(self.conf._overrides.wine_args or [])
+        )
+
     def run_winetricks(self):
         wine.run_winetricks(self)
```

Comment:
How does the CLI ensure winetricks is installed in this mode? Personally I'd be fine with (python psudo-code):

```py
from typing import Never

def run_winetricks() -> Never
  if appimage:
    winetricks_binary = [appimage, "winetricks"]
  else if found_in_path("winetricks"):
     winetricks_binary = ["winetricks"]
  else:
    app.exit("winetricks not installed")
  cmd_run(winetricks) # ideally capture stdin/stdout
  # If the process passes
  os.exit(0)
```

Reply:
I'd be happy to drop the `--winetricks` subcommand, but I find the `--wine` one very useful, and I don't think it adds too much additional maintenance overhead.

Reply:
I'd be happy to drop the `--winetricks` subcommand, but I find the `--wine` one very useful, and I don't think it adds too much additional maintenance overhead.

Reply:
I've opted to remove remaining winetricks code. I've run a couple of test installs and TUI commands, but I'd like to test a bit more before asking for another review.